prow/cmd/branchprotector/protect_test.go

```diff
@@ -597,28 +597,6 @@ branch-protection:
 				},
 			},
 		},
-		{
-			name:     "do not unprotect unprotected",
```

Comment:
We definitely need this unit test

Reply:
Yeah I agree, but the current implementation does not work